website/www/site/content/en/contribute/_index.md

```diff
@@ -98,27 +109,52 @@ sudo apt-get install \
    docker-ce
 ```
 
-On some systems (like Ubuntu 20.04) these need to be installed also
+2. On some systems, like Ubuntu 20.04, install these:
+
 ```
 pip3 install grpcio-tools mypy-protobuf
 ```
 
-You also need to [install Go](https://golang.org/doc/install).
-
-Once Go is installed, install goavro:
+3. If you you develop in GO:
+    1. Install [Go](https://golang.org/doc/install).
+    1. Check BEAM repo is in: `$GOPATH/src/github.com/apache/`
+    1. At the end, it should look like this: `$GOPATH/src/github.com/apache/beam`
+4. Once Go is installed, install goavro:
 
 ```
 $ export GOPATH=`pwd`/sdks/go/examples/.gogradle/project_gopath
 $ go get github.com/linkedin/goavro
 ```
 
-gLinux users should configure their machines for sudoless Docker.
+**Important**: gLinux users should configure their machines for sudoless Docker.
+
+##### Automated script for Linux and macOS
+
+You can install these in a Debian-based distribution for Linux or macOs using the [local-env-setup.sh](https://github.com/apache/beam/blob/master/local-env-setup.sh) script, which is part of the Beam repo. It contains:
```

Comment:
This file doesn't exist. Is it being added in another PR? If so, could we add this section as part of that PR instead?

Reply:
Yes, added to [https://github.com/apache/beam/pull/14584](https://github.com/apache/beam/pull/14584 ) because [https://issues.apache.org/jira/browse/BEAM-11608](https://issues.apache.org/jira/browse/BEAM-11608) intended to solve that. This PR only is intended for documentation improvements

Reply:
Got it, thank you. 

Note that we build and deploy the Beam website continuously (every few hours) from this source. So shortly after we merge this PR the website will be updated accordingly. So if this PR is going to add a reference to local-env-setup.sh, I don't want to merge it until that script is actually available. We can either:
- Wait to merge this PR until #14584 is merged, or
- Split out the parts that reference local-env-setup.sh into a different PR, so we can go ahead and merge this one

Either way is fine with me, just let me know which you'd prefer

Reply:
Hi, @tysonjh based on Brian's suggestions, which approach would like us to follow?

Reply:
I'm in favor of waiting to merge.